pymatgen/command_line/bader_caller.py

```diff
@@ -231,6 +225,62 @@ def summary(self):
 
         return summary
 
+    @classmethod
```

Comment:
I think this makes sense, but if you're adding `.from_path()` here, I'd delete the `bader_analysis_from_path()` standalone function. No point keeping both!